### ISSUE TITLE
usage of output_dir is not as intended

### ISSUE THREAD

**Issue by gregcaporaso** [allenlab/QIIME2_16S_ASV_protocol#1]
Hello, I noticed this comment in your protocol:

```
# Files don't actually get put in output-dir
```

If you specify the `--output-dir` option, and outputs whose paths are not specified with `--o-` options will be put in the specified output directory with a default name. So, think of the `--output-dir` as the default location where outputs will be written, which you can then override for none, some, or all of the outputs by providing their ``--o-` options. 

Some variants on your command and their results are as follows:

```bash
# write all three outputs to the asvs directory with a default file names
qiime dada2 denoise-paired \
    --i-demultiplexed-seqs paired-end-demux-trimmed.qza \
    --p-n-threads 0 \
    --p-trunc-q 2 \
    --p-trunc-len-f 219 \
    --p-trunc-len-r 194 \
    --p-max-ee-f 2 \
    --p-max-ee-r 4 \
    --p-n-reads-learn 1000000 \
    --p-chimera-method pooled \
    --output-dir ./asvs

# write 2 of 3 outputs to the asvs directory with a default file name, and put the other in a specified location
qiime dada2 denoise-paired \
    --i-demultiplexed-seqs paired-end-demux-trimmed.qza \
    --p-n-threads 0 \
    --p-trunc-q 2 \
    --p-trunc-len-f 219 \
    --p-trunc-len-r 194 \
    --p-max-ee-f 2 \
    --p-max-ee-r 4 \
    --p-n-reads-learn 1000000 \
    --p-chimera-method pooled \
    --output-dir ./asvs/ \
    --o-table some-other-directory/table-dada2.qza 

# write all three outputs to specified locations 
qiime dada2 denoise-paired \
    --i-demultiplexed-seqs paired-end-demux-trimmed.qza \
    --p-n-threads 0 \
    --p-trunc-q 2 \
    --p-trunc-len-f 219 \
    --p-trunc-len-r 194 \
    --p-max-ee-f 2 \
    --p-max-ee-r 4 \
    --p-n-reads-learn 1000000 \
    --p-chimera-method pooled 
    --o-table table-dada2.qza \
    --o-representative-sequences rep-seqs-dada2.qza \
    --o-denoising-stats stats-dada2.qza
```

**Comment by rhlampe** [comment]
Great! Thank you for looking this over and clarifying this!

**Comment by rhlampe** [comment]
Fixed with commit [70286c4e78a0e6e2b5302c6092ea6ee776f01db5](https://github.com/allenlab/QIIME2_16S_ASV_protocol/commit/70286c4e78a0e6e2b5302c6092ea6ee776f01db5)
